api/src/models/priceHistory.model.ts

```diff
@@ -1,19 +1,19 @@
 import { model, Schema, Document } from 'mongoose';
 import { PriceHistoryData } from '@interfaces/priceHistory.interface'; 
+import { PokemonCardModel } from './cards.model';
 
 const PriceHistorySchema: Schema = new Schema({
-  month: {
-    type: Number, 
-    required: true, 
-  }, 
-  day: {
-    type: Number, 
-    required: true, 
-  }, 
-  year: {
-    type: Number, 
-    required: true, 
-  }, 
+  // Not sure if id is required 
+  // I think it is because the cards references PriceHistorySchema
+  id: {
```

Comment:
You may not need this as it'll be automatically added with Mongoose. You should check the documentation.